firstname_lastname.txt

```diff
@@ -0,0 +1 @@
+## Contributors: Iana Strigunova
```

Comment:
The name of the file is supposed to be _your_ first name and last name, not the placeholder "firstname_lastname" :) 

Also, please add it to the `## Contributors` section in the README.md file.